amplify/functions/send-email/email-template.ts

```diff
@@ -8,12 +8,14 @@ const encoder = (rawText: string) => encode(rawText).replaceAll('\n', '<br />');
 export const emailTemplate = (
   templateId: string,
   templateName: string,
-  templateMessage: string
+  templateMessage: string,
+  templateSubjectLine?: string | null
```

Comment:
this is allowing for separate cases for when it's passed in as null and when it's not passed in - is this needed?

Reply:
The issue is we have is when we save the template and as a result get the returned template from amplify. The subjectLine for non email templates this value will be null.

But it probably doesn't make sense for it to ever be optional in this instance. Will update